SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
-    path_to_new_waveform_h5 = ''.join(path_to_new_waveform_h5).replace(w_m.descriptor_string + '_', '')
```

Comment:
But it *probably can* handle the input data.  There's no way to tell if the user incorrectly named a file and thus wanted a different type of transformation, or if the user correctly named the file and wanted the type of transformation that `scri` gave it.  If the output file is named in some unexpected way, the user should notice -- despite the fact that there's no way for `scri` to know what s/he wanted.